{{cookiecutter.service_name}}/tests/run_tests.sh

```diff
@@ -2,7 +2,9 @@
 
 ###
 # Script requirments:
-#  apt-get install -y python-yaml virtualenv git
+#apt-get install -y python-yaml virtualenv git
+which virtualenv &> /dev/null || apt-get install -y virtualenv
```

Comment:
remove, possibly add install to Makefile
make it multiplatform (like centos way)